### PR TITLE
Add firefox/welcome/page2.lang

### PR DESCRIPTION
**Pull request by craigcook**
Page demo:  https://bedrock-demo-craigcook.oregon-b.moz.works/firefox/welcome/2/
Bedrock pull request: https://github.com/mozilla/bedrock/pull/7869

### CODE REVIEW

**Comment by craigcook** [comment]
Sources updated in https://github.com/mozilla-l10n/langchecker/pull/931

File: en-US/firefox/welcome/page2.lang [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# "Pocket" is a product brand name and shouldn't be translated.
+;Activate Pocket
+Activate Pocket
+
+
+;Save content from everywhere
+Save content from everywhere
+
+
+;Grab articles, videos, and links from any website by clicking the Pocket icon in your browser toolbar.
```
Comment (peiying2) [inline en-US/firefox/welcome/page2.lang]: Pocket is mentioned in most of the strings in this page. Do you want to add another comment for this Pocket? :).

File: en-US/firefox/welcome/page2.lang [hunk 2]
```diff
@@ -0,0 +1,49 @@
+## NOTE: Demo https://bedrock-demo-craigcook.oregon-b.moz.works/firefox/welcome/2/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/welcome/2/
+
+
+# HTML page title
+# "Pocket" is a product brand name and shouldn't be translated
+;Pocket - Save news, videos, stories and more
```
Comment (peiying2) [inline en-US/firefox/welcome/page2.lang]: a, b, c, and more? An extra comma proceeding `and`?

Comment (craigcook) [reply en-US/firefox/welcome/page2.lang]: House style guide is not to use a serial comma so this is correct.

File: en-US/firefox/welcome/page2.lang (hunk 1 above)
Comment (craigcook) [reply en-US/firefox/welcome/page2.lang]: Just to be thorough!

**Comment by craigcook** [comment]
The demo page is being sent to stakeholders for a final review so I'm going to hold off merging this until tomorrow, just in case something changes.

**Pull request merged**